docs/Reference/Config-Items.md

```diff
@@ -16,6 +20,26 @@ Network configuration items are specified in the Hyperledger Besu genesis file.
 | `evmStackSize`      | Maximum stack size. Specify to increase the maximum stack size in private networks with very complex smart contracts. Default is `1024`. |
 
 
+## Genesis Block Parameters  
+ 
+The purpose of some genesis block parameters parameters varies depending on the consensus protocol 
+(Ethash, [Clique](../HowTo/Configure/Consensus-Protocols/Clique.md), or [IBFT 2.0](../HowTo/Configure/Consensus-Protocols/IBFT.md)). 
+These parameters include: 
+
+* `difficulty`
+* `extraData`
+* `mixHash`
+ 
+Genesis block parameters with the same purpose across all consensus protocols are described below. 
+
+| Item                | Description                                                                                                                              |
+|---------------------|-:----------------------------------------------------------------------------------------------------------------------------------------|
+| `coinbase`          | Address to which mining rewards are paid. Can be any value in the genesis block . 
```

Comment:
Perhaps we could advertise the usually used `0x0000000000000000000000000000000000000000` value so users don't wonder which one to use as they can put whatever they want.

Reply:
Done